webapp/src/components/link_tooltip/tooltip.css

```diff
@@ -0,0 +1,121 @@
+:root {
+    --light-gray: #6a737d;
+    --gray: #586069;
+    --dark-gray: #24292e;
+    --blue: #274466;
+    --light-blue: #eff7ff;
+}
+
+@media (min-width: 544px) {
+    .gitlab-tooltip--large {
+        min-width: 320px;
+    }
+}
+
+/* Gitlab Tooltip */
+.gitlab-tooltip .box {
+    background-color: white;
+    box-shadow: 0 1px 15px rgba(27,31,35,.15);
+    position: relative;
+    width: 360px;
+}
+
+.gitlab-tooltip--bottom-left, .gitlab-tooltip--top-left {
+    left: -9px;
+    margin-left: 0;
+}
+
+/* Header */
+.gitlab-tooltip .header {
+    color: var(--light-gray);
+    font-size: 12px;
+}
+
+.gitlab-tooltip .header a {
+    text-decoration: none;
+    color: var(--gray);
+    display: inline-block;
+}
+
+/* Body */
+.gitlab-tooltip .body > span {
+    line-height: 1.25;
+}
+
+/* Info */
+.gitlab-tooltip .tooltip-info {
+    line-height: 1.25;
+    display: flex;
+    flex-direction: column;
+}
+.gitlab-tooltip .tooltip-info > a, .gitlab-tooltip .tooltip-info > a:hover {
+    display: block;
+    text-decoration: none;
+    color: var(--dark-gray);
+}
+.gitlab-tooltip .tooltip-info > a span {
+    color: var(--light-gray);
+}
+
+.gitlab-tooltip .tooltip-info h5 {
+    font-weight: 600;
+    font-size: 14px;
+    display: inline;
+}
+
+.gitlab-tooltip .tooltip-info .markdown-text {
+    -webkit-hyphens: auto;
+    -ms-hyphens: auto;
+    hyphens: auto;
+    max-height: 150px;
+    line-height: 1.25;
+    overflow: hidden;
+    font-size: 12px;
+}
+
+.gitlab-tooltip .tooltip-info .markdown-text::-webkit-scrollbar {
```

Comment:
`::-webkit-scrollbar` will not work on some browsers like Firefox

pls see here the list of support https://caniuse.com/?search=%3A%3A-webkit-scrollbar, if there is any browser that you want to support but is not supported check what's the alternative for them

Reply:
Discussed.